inspectit-oce-core/src/main/resources/config/default.yml

```diff
@@ -36,6 +36,51 @@ inspectit:
 
   # settings regarding the instrumentation performed on the target application
   instrumentation:
+
+#    # specifying instrumentation scopes which are used by rules for targeting classes and methods
+#    # please see the documentation for detailed information about this configuration item
+#    scopes:
+#      # the id of the following defined scope element - this example scope targets the ArrayList's contains method
+#      example-list-scope:
+#        # the targeted type - classes which should be instrumented have to match ALL of the specified elements within the type scope
+#        type-scope:
+#          # interfaces which have to be implemented
+#          interfaces:
+#            - name-pattern: "java.util.List"
+#              matcher-mode: "EQUALS_FULLY"
+#          # the superclass which has to be extended
+#          superclass:
+#            name-pattern: "java.util.AbstractList"
+#            matcher-mode: "EQUALS_FULLY"
```

Comment:
you can actually omit the EQUALS_FULLYs here, right?

Reply:
Yes, just as an example including all available attributes :)